gnovm/pkg/gnolang/values_string.go

```diff
@@ -298,24 +340,28 @@ func (tv *TypedValue) Sprint(m *Machine) string {
 		return res[0].GetString()
 	}
 
-	return tv.ProtectedSprint(newSeenValues(), true)
+	return tv.ProtectedSprint(m, newSeenValues(), true)
 }
 
-func (tv *TypedValue) ProtectedSprint(seen *seenValues, considerDeclaredType bool) string {
+func (tv *TypedValue) ProtectedSprint(m *Machine, seen *seenValues, considerDeclaredType bool) string {
+	m.GasMeter.ConsumeGas(OpCPUAssign, CPUCYCLES)
+
+	defer func() { m.GasMeter.ConsumeGas(OpCPUAssign, CPUCYCLES) }()
```

Comment:
I guess the "OpCPUAssign" here borrowed from machine opcode is an approximate value, right? it won't be accurate without benchmarking. I think we can put it here and make some meaningful comments so we can benchmark in future.

Reply:
updatd